web/sbom.json

```diff
@@ -8235,7 +8235,7 @@
       "name": "ms",
       "version": "2.0.0",
       "bom-ref": "ms@2.0.0",
-      "description": "Tiny milisecond conversion utility",
+      "description": "Tiny millisecond conversion utility",
```

Comment:
We shouldn't fix this file, this is created by other project information. it is auto generated.😄